src/Symfony/Component/Notifier/Chatter.php

```diff
@@ -33,7 +33,7 @@ public function __construct(TransportInterface $transport, MessageBusInterface $
     {
         $this->transport = $transport;
         $this->bus = $bus;
-        $this->dispatcher = LegacyEventDispatcherProxy::decorate($dispatcher);
+        $this->dispatcher = class_exists(LegacyEventDispatcherProxy::class) && is_a(LegacyEventDispatcherProxy::class, EventDispatcherInterface::class, true) ? LegacyEventDispatcherProxy::decorate($dispatcher) : $dispatcher;
```

Comment:
My intention here was that I wanted to keep the notifier compatible with Symfony 4.4. This line should detect if we still have a Symfony 4 dispatcher. In that case, we would still apply the decorator.